src/core/utils.js

```diff
@@ -123,14 +123,18 @@ export function loadAddonIfNeeded(
   return _refreshAddon({ addonSlug: slug, apiState: state.api, dispatch });
 }
 
-export function isAllowedOrigin(urlString, { allowedOrigins = [config.get('amoCDN')] } = {}) {
+export function isAllowedOrigin(urlString, {
+  allowedOrigins = [config.get('amoCDN')],
+  allOriginsAllowed = config.get('allOriginsAllowed'),
```

Comment:
Oops, yeah sorry should have removed it. Probably why the tests are failing too (lint). Will send a patch in a sec.